src/highlight.js

```diff
@@ -247,10 +247,29 @@ const HLJS = function(hljs) {
     }
 
     /**
-     * @param {Mode} mode - new mode to start
+     * @param {Mode} mode
```

Comment:
```suggestion
     * @param {CompiledMode} mode
```

The only time this function is called, it'll always be a `CompiledMode`